setup.py

```diff
@@ -210,6 +217,34 @@
 
 import platform
 
+
+def _get_package_path(package_name):
+    loader = pkgutil.find_loader(package_name)
+    if loader:
+        # The package might be a namespace package, so get_data may fail
+        try:
+            file_path = loader.get_filename()
+            return os.path.dirname(file_path)
+        except AttributeError:
+            pass
+    return None
+
+
+BUILD_LIBTORCH_WHL = os.getenv("BUILD_LIBTORCH_WHL", "0") == "1"
+BUILD_PYTORCH_USING_LIBTORCH_WHL = os.getenv("BUILD_PYTHON_ONLY", "0") == "1"
+
+
+# set up appropriate env variables
+if BUILD_LIBTORCH_WHL:
+    # Set up environment variables for ONLY building libtorch.so and not libtorch_python.so
+    # functorch is not supported without python
+    os.environ["BUILD_FUNCTORCH"] = "OFF"
+
+
+if BUILD_PYTORCH_USING_LIBTORCH_WHL:
+    os.environ["BUILD_LIBTORCHLESS"] = "ON"
```

Comment:
Can we use only 1 var BUILD_PYTORCH_USING_LIBTORCH_WHL everywhere instead of 
```
BUILD_PYTORCH_USING_LIBTORCH_WHL
BUILD_LIBTORCHLESS
```

Reply:
It's two separate builds so not at the moment.

Reply:
It looks like these PR introduce following new flags:
```
BUILD_LIBTORCH_WHL
BUILD_PYTORCH_USING_LIBTORCH_WHL
BUILD_LIBTORCHLESS
BUILD_PYTHON_ONLY
```

is it possible to define what each flag is used for? and maybe use fewer of the flags ?
I see following:
```
BUILD_PYTORCH_USING_LIBTORCH_WHL = os.getenv("BUILD_PYTHON_ONLY", "0") == "1"
...
 if BUILD_PYTORCH_USING_LIBTORCH_WHL:
        os.environ["BUILD_LIBTORCHLESS"] = "ON"
```
Does it mean we can use 
```
BUILD_PYTHON_ONLY = os.getenv("BUILD_PYTHON_ONLY", "0") == "1"
```
?